bmarlow/Week 3/mailroom.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+#build our initial dictionary
+dictionary = {
+	'Ricky': '1000',
+	'Bobby': '1100,2200',
+	'Hansel': '5000,6000,7000',
+	'Mr. Burgandy': '1,2,3,4',
+	'Austin': '123,4567,78,99,100',
+	'Dale': '14,234,878,345,345,234'
+}
+
+#we want it to be pretty, that means wiping junk off the screen
+def clear():
+	'''cross-platform screen clear, for prettiness'''
+	os.system('cls' if os.name == 'nt' else 'clear')
+
```

Comment:
I'm too lazy to write code in a VM, so its on my windows box.  Wanted you Mac guys to have a pretty screen too :-)